Client.xcodeproj/project.pbxproj

```diff
@@ -2902,6 +2902,7 @@
 				TargetAttributes = {
 					2827315D1ABC9BE600AA1954 = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = 43AQ936H96;
```

Comment:
This change, and any of the changes below should not be committed. The defaults in the project file should really just work? No? Are you using multiple dev accounts on your machine? Maybe we need to be more specific in the defaults then so that they default to Mozilla?